javacomparator/DiffGenerator.java

```diff
@@ -0,0 +1,457 @@
+import org.apache.commons.io.FileUtils;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TreeSet;
+
+/**
+ * DiffGenerator class finds the Difference between files in two folders
+ * using the class FileComparator and Generates the Diff report in 3 different cases
+ * Individual comparision report, Global run Summary and All diff compilation
+ */
+public class DiffGenerator {
+    String pathOrigFile;
```

Comment:
You should have completely meaningful names for the variables -example: pathOriginalFile instead of path OrigFile